src/plugins/chart_expressions/expression_metric/public/components/metric_component.tsx

```diff
@@ -117,12 +117,25 @@ class MetricVisComponent extends Component<MetricVisComponentProps> {
 
     return (
       <MetricComponent
+        autoScaleParams={
+          this.props.visParams.metric.autoScale && this.props.visParams.metric.colorFullBackground
```

Comment:
I'd move out this condition to a separate function and reuse it over these components in case it will become more complex. And also you'll not need those comments, because the name of the function will be descriptive.

Reply:
done